ecl/eclcc/eclcc.cpp

```diff
@@ -1976,7 +1976,12 @@ bool EclCC::parseCommandLineOptions(int argc, const char* argv[])
         else if (strcmp(arg, "-internal")==0)
         {
             outputSizeStmts();
-            testHqlInternals();
+            int error = testHqlInternals() + testReservedWords();  //NOTE: testReservedWords() depends on testHqlInternals() so must be be called after.
+            // report test result
+            if (error)
+                printf("%d error%s found!\n", error, error<=1?"":"s");
+            else
+                printf("No errors\n");
```

Comment:
So long as success returns 0 and failure returns something else the exact values aren't that significant